README.md

```diff
@@ -64,45 +64,91 @@ Please note that as the plugin continues to take shape, there might be breaking
+Perhaps someone who's more of a Composer expert could lend some advise?:
+
 
-NOTE: You'll want the test database to be a true test database, not a database with valuable, existing information, as 
-the tests will create new data and clear out data, and you don't want to cause issues with a database you're actually 
-using for projects.
+- `rm -rf composer.lock vendor` to remove all composer dependencies and the composer lock file
+- `composer require lucatume/wp-browser --dev` to install the Codeception WordPress deps
+- `vendor/bin/codecept run` to run all the codeception tests
+    - You can specify which tests to run like: 
+        - `vendor/bin/codecept run wpunit`
+        - `vendor/bin/codecept run functional`
+        - `vendor/bin/codecept run unit`
+        - `vendor/bin/codecept run acceptance`
+        
+### Generating Code Coverage
+You can generate code coverage for tests by passing `--coverage`, `--coverage-xml` or `--coverage-html` with the tests. 
 
-DEBUGGING: If you have run this command before in another branch you may already have a local copy of WordPress downloaded in your `/private/tmp` directory. If this is the case, please remove it and then run the install script again. Without removing this you may receive an error when running phpunit.
+- `--coverage` will print coverage info to the screen
+- `--coverage-xml` will save an XML file that can be used by services like Coveralls or CodeCov
+- `--coverage-html` will save the coverage report in an HTML file that you can browse. 
 
-### Running the Tests and Checking Coverage
-Now that we have the prerequisites installed we can run the tests and check our code coverage. The results of our tests will appear on the command line, however our code coverage information is stored in the coverage directory in the form of HTML. This HTML is generated by phpunit when it runs the tests by passing the `--coverage-html` flag.
+The coverage details will be output to `/tests/_output`
+        
+#### Local Environment Config for Codeception Tests
 
-`phpunit --coverage-html ./coverage`
+You may have different local environment configuration than what Travis CI has to run the tests, such as database username/password.
 
-The easiest way to check your code coverage is to drag, or right click and open, the `index.html` file from the coverage directory (`wp-graphql/coverage/index.html`) in your browser. This should render a dashboard that will allow you to view the code coverage of all files in the WPGraphQL repo.
+In the `/tests` directory you will find `*.suite.dist.yml` config files for each of the codeception test suites. 
+
+You can copy those files and remove the `.dist` from the filename, and that file will be loaded locally _before_ the `.dist` file.
+
+For example, if you wanted to update the `dbPassword` for your local tests, you could copy `wpunit.suite.dist.yml` to `wpunit.suite.yml` and update the `dbPassword` value to what your local password is.
```

Comment:
@jasonbahl, can you swap this line out for something like this:

```
For example, if you wanted to update the `dbName` or `dbPassword` for your local tests, you could copy `wpunit.suite.dist.yml` to `wpunit.suite.yml` and update the `dbName` or `dbPassword` value to reflect your local database and password.
```

Reply:
Also, could you move this whole block to sit below the `Installing the Test Suite` section?